chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/AnnouncementManagerImpl.java

```diff
@@ -92,12 +92,12 @@ public void setAnnouncementDAO(AnnouncementDAO announcementDAO) {
 		this.announcementDAO = announcementDAO;
 	}
 
-	@PreAuthorize("hasRole('ROLE_ADMIN')")
+	@PreAuthorize("hasRole('ROLE_ADMIN') or hasRole('ROLE_ONC_STAFF')")

```

Comment:
I don't think ONC_STAFF needs access to manage Announcements. Is that what's being done here? This isn't the Announcement Activity stuff, right?